webapp/src/components/EditEstatePage/EstateSelect/EstateSelect.css

```diff
@@ -7,55 +7,34 @@
   font-size: 15px;
 }
 
-.EstateSelect .select-title {
-  margin-bottom: 5px;
-  font-weight: bold;
-}
-
-.EstateSelect > .Loading {
-  font-size: 10px;
-  top: 35%;
-}
-
-.EstateSelect .description {
-  color: var(--gray);
-}
-
 .EstateSelect .estate-selection {
   margin: 30px 0;
-  font-size: 15px;
 }
 
-.EstateSelect .EstateActions {
-  text-align: left;
-  margin-top: 30px;
-}
-
-.EstateSelect .estate-selection .parcels-included.column {
-  display: flex;
-  flex-wrap: wrap;
+.EstateSelect .estate-selection .header {
+  display: inline-block;
 }
 
 .EstateSelect .estate-selection .parcels-included-description {
-  width: 100%;
   font-size: 18px;
-  margin: 15px 0 0;
   color: var(--gray);
 }
 
-.EstateSelect .parcels-included .ParcelCoord {
-  margin-right: 16px;
-  margin-top: 20px;
+.EstateSelect .estate-selection .dissolve-button {
+  padding-top: 6px;
+  float: right;
 }
 
-.EstateSelect .parcels-included-headline.column {
-  margin-top: 20px;
-  text-align: right;
-}
-
-.tx-label {
+.EstateSelect .tx-label {
   text-align: left;
   font-size: 14px;
   margin-top: 15px;
   color: var(--text);
 }
+
+@media (max-width: 475px) {
+  .EstateSelect .estate-selection .dissolve-button {
+    padding-bottom: 6px;
+    float: none;
```

Comment:
Question: Did you want to use `clear: both` here?

Reply:
Nope! I wanted to remove the `float: right` from the button.
If I clear the float I'd look like this:

![image](https://user-images.githubusercontent.com/692077/49143675-d2abfd00-f2fb-11e8-84b9-cae6d9e72404.png)

but I want this:

![image](https://user-images.githubusercontent.com/692077/49143683-db9cce80-f2fb-11e8-8ac6-2cbe7d28b5d0.png)